### PR TITLE
Remove remaining bits of OPTE External IP hack.

### PR DESCRIPTION
**Pull request by luqmana**
Pull in updated OPTE and with that get rid of remaining bits of this hack!

### CODE REVIEW

File: tools/create_virtual_hardware.sh [hunk 1]
```diff
@@ -138,21 +138,7 @@ function enable_softnpu {
...
-
-    grep "ext_ip_hack = 0;" /kernel/drv/xde.conf || {
-        echo "failed to disable ext_ip_hack"
-        exit 1
-    }
-    success "ext_ip_hack disabled"
-}
-
 ensure_run_as_root
-ensure_ext_ip_hack_disabled
```
Comment (internet-diglett) [inline tools/create_virtual_hardware.sh:155]: This function ensures that on any machine that is being upgraded from an older version of Omicron (pre-OPTE) that the hack gets disabled. I don't know if there are still machines / environments out there with the hack, but maybe we want to leave this here until we know for sure that there aren't any opte-hacked users still around?

**Review by internet-diglett — APPROVED** [review]
Left one comment, otherwise LGTM!

File: tools/create_virtual_hardware.sh (hunk 1 above)
Comment (luqmana) [reply tools/create_virtual_hardware.sh:155]: Well the `ext_ip_hack` config should no longer exist once they upgrade to this newer xde build.

Comment (internet-diglett) [reply tools/create_virtual_hardware.sh:155]: Ah, so it won't even look for the value anymore. Perfect!

**Pull request merged**